stunnel/stunnel.mli

```diff
@@ -53,10 +54,15 @@ val connect :
   string -> int -> t
 
 (** Disconnects from stunnel and cleans up *)
-val disconnect : ?wait:bool -> ?force:bool -> t -> unit
+val disconnect : ?wait:bool -> ?force:bool -> ?verbose:bool -> t -> unit
```

Comment:
Discussed the motivation to have logging done via a flag rather than relying on syslog logging levels and why we're using `info` logging level despite the `versbose` flag and it all seems good. Basically we don't want to log every disconnect as that would be way too noisy. Just the ones as a result of changing the permissiveness.